content/docs/user-guide/merge-conflicts.md

```diff
@@ -0,0 +1,137 @@
+# Merge Conflicts
+
+Sometimes multiple members of a team might worn on the the same DVC-tracked
+data. And when the time comes to combine their changes, merge conflicts can
+arise in Git, which need to be resolved.
+
+## dvc.yaml
+
+Conflicts here are no different from what we would see in source code. See
+[Git Merging](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging).
+
+```yaml
+stages:
+  prepare:
+    cmd: python src/prepare.py data/data.xml
+    deps:
+< < < < < < < HEAD
+    - data/big.xml
+= = = = = = =
+    - data/small.xml
+> > > > > > > branch
+    - src/prepare.py
+    params:
+    - prepare.seed
+    - prepare.split
+    outs:
+    - data/prepared
+```
+
+## dvc.lock
+
+There's no need to resolve lockfile conflicts manually. You can safely delete
+this file and then use `dvc repro` after merging `dvc.yaml` to regenerate this
+file.
+
+## \*.dvc
+
+There are three three main variations in the structure of these files, that
+differ by the command that has generated them:
+
+### Simple tracking (add)
+
+In `.dvc` files generated by `dvc add`, you'll get something that looks like:
+
+```yaml
+outs:
+< < < < < < < HEAD
+- md5: a304afb96060aad90176268345e10355
+= = = = = = =
+- md5: 35dd1fda9cfb4b645ae431f4621fa324
+> > > > > > > branch
+  path: data.xml
+```
+
+If you decide to just pick one of the versions, leave that md5 and delete the
+other one:
+
+```yaml
+outs:
+  - md5: 35dd1fda9cfb4b645ae431f4621fa324
+    path: data.xml
+```
+
+But if you want to do actually merge data files (or directories) from both
+versions, then you can follow this process:
+
+1. Run `dvc checkout data.xml` on both `HEAD` and `branch`;
+2. Copy the data into temporary locations (e.g. `data.xml.head` and
+   `data.xml.branch`);
+3. Merge it by-hand;
+4. Finally, run `dvc add data.xml` to overwrite the conflicted `.dvc` file.
+
+#### Append-only directories
+
+If you have an "append-only" dataset, where people only add new
+files/directories, DVC provides a so-called
+[merge-driver](https://git-scm.com/docs/git-merge#Documentation/git-merge.txt-mergeltdrivergtname)
+that can automatically resolve Git conflicts for you. To use it, first set it up
+in your Git repo:
+
+```dvc
+git config merge.dvc.name "DVC merge driver"
+git config merge.dvc.driver "dvc git-hook merge-driver --ancestor %O --our %A --their %B"
```

Comment:
Should all these be `'` single quotes @efiop ? Or is it all the same.

Reply:
Doesn't matter in this case for *nix, but might indeed be problematic for windows. Good catch! Adjusted in https://github.com/iterative/dvc.org/commit/7a65f1119598673eb88e3d267cd421f589b3ccdd . Thank you! :pray:

Reply:
Thanks!